src/ZipkinBundle/Components/HttpClient/HttpClient.php

```diff
@@ -2,176 +2,18 @@
 
 namespace ZipkinBundle\Components\HttpClient;
 
-use Zipkin\Tracer;
-use Zipkin\SpanCustomizerShield;
-use Zipkin\SpanCustomizer;
-use Zipkin\Span;
-use Zipkin\Propagation\TraceContext;
-use Zipkin\Propagation\Map;
 use Zipkin\Instrumentation\Http\Client\Parser;
-use Zipkin\Instrumentation\Http\Client\HttpClientTracing;
-use TypeError;
-use Throwable;
-use Symfony\Contracts\HttpClient\ResponseStreamInterface;
-use Symfony\Contracts\HttpClient\ResponseInterface;
 use Symfony\Contracts\HttpClient\HttpClientInterface;
-use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
-use Symfony\Component\HttpClient\Response\ResponseStream;
 
 final class HttpClient implements HttpClientInterface
 {
-    /**
-     * @var HttpClientInterface
-     */
-    private $delegate;
-
-    /**
-     * @var Tracer
-     */
-    private $tracer;
-
-    /**
-     * @var callable
-     */
-    private $injector;
+    use HttpClientTrait;
```

Comment:
Quick question: is this necessary?

Reply:
Nope, it was a try to make this flexible for supporting both <8.0 and >8.0, I'll remove it.